### PR TITLE
Aligned OpenStack content to the new expressions and string constants syntax 

### PR DESCRIPTION
**Pull request by mihaitusa**
Align test section with our latest syntax conventions - flavor, images, serveractions && corresponding tests.

### CODE REVIEW

**Comment by mihaitusa** [comment]
@oritstone  Please review it. Thanks!

File: content/io/cloudslang/openstack/blockstorage/create_volume.sl [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        navigate:
+          SUCCESS: create_volume
+          GET_AUTHENTICATION_TOKEN_FAILURE: GET_AUTHENTICATION_TOKEN_FAILURE
+          GET_TENANT_ID_FAILURE: GET_TENANT_ID_FAILURE
+          GET_AUTHENTICATION_FAILURE: GET_AUTHENTICATION_FAILURE
+
+    - create_volume:
+        do:
+          rest.http_client_post:
+            - url: "${'http://' + host + ':' + blockstorage_port + '/v2/' + tenant_id + '/volumes'}"
```
Comment (Bonczidai) [inline content/io/cloudslang/openstack/blockstorage/create_volume.sl]: no need for quotes here -  please do a search on this changelist after `"$` and remove the quotes from all the places where it is not necessary

File: content/io/cloudslang/openstack/blockstorage/create_volume.sl [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          - status_code
+        navigate:
+          SUCCESS: get_volume_id
+          FAILURE: CREATE_VOLUME_FAILURE
+
+    - get_volume_id:
+        do:
+          json.get_value:
+            - json_input: ${return_result}
+            - json_path: ["'volume'", "'id'"]
```
Comment (Bonczidai) [inline content/io/cloudslang/openstack/blockstorage/create_volume.sl]: I think it should be only one quotes - please verify (all the places)

**Comment by Bonczidai** [comment]
much neater now

**Pull request merged**